### ISSUE TITLE
Number being cast to string causes  domain filter to be applied incorrectly

### ISSUE THREAD

**Issue by tiaanduplessis** [vettloffah/odoo-await#8]
Thank you for maintaining this package. When trying to apply a domain filter that has an integer value e.g.

```js
odoo.searchRead(`hr.job`, {  company_id: 1 }, fields, params)
```

The typecasting to a string of the value happening (https://github.com/vettloffah/odoo-await/blob/master/lib/index.js#L252) leads to incorrect results being returned. 

When removing this typecasting:

```js
domainArray.push([`${key}`, '=', value]);
```

I receive the correct results.

If you are currently unable to assist with an update, I will be more than happy to create a PR.

**Comment by vettloffah** [comment]
Ah!

Nice catch.

I'll work this in to the next update which should be today or tomorrow.

**Comment by tiaanduplessis** [comment]
Awesome! Thanks for the quick response :+1:

**Comment by vettloffah** [comment]
Resolved with version 2.1.1 (somehow missed this in the last update)
